packages/financial-templates-lib/src/clients/OptimisticOracleClient.ts

```diff
@@ -51,10 +84,12 @@ export class OptimisticOracleClient {
     public readonly web3: Web3,
     oracleAddress: string,
     votingAddress: string,
-    public readonly lookback = 604800 // 1 Week
+    public readonly lookback: number = 604800, // 1 Week
+    public readonly oracleType: OptimisticOracleType = "OptimisticOracle"
```

Comment:
making this an `enum` of the two possible types is probs better.

Reply:
where this is used is in js and a string is probs better there...I think that's ok just please add validation where it's read in within the `index` file and in this file that it is one of the two possible valid options and error if not.

Reply:
Made it an enum!